src/plot_recipes.jl

```diff
@@ -306,7 +306,7 @@ julia> plot(Singleton([0.5, 1.0]));
 end
 
 """
-    plot_singleton(arr::Vector{<:AbstractSingleton}; ...)
+    plot_singleton(arr::Vector{S}; ...) where {S<:AbstractSingleton}
```

Comment:
these changes are motivated by style only? for me both are OK; if i had to choose, the one in the left is a bit shorter but maybe the notation in the right is more clean.. anyway 😄

Reply:
No, I did not find a solution for these cases other than introducing a (useless) type parameter. But I rather wanted to close that issue, so I did that change. I kept them in a separate commit if we do not want to go that way (after all, it is weird to change the code just for documentation). But I do not find it that bad.

Reply:
so simple cases like `function f(Vector{<:LazySet}) return 1 end` where giving problems?

Reply:
Yes, they are not found by `Documenter` (at least I did not find a way).